faq/databases-for-redis.mdx

```diff
@@ -0,0 +1,74 @@
+
+## Why is it ideal for Cache usage?
+
+Based on the **Re**mote **Di**ctionary **S**erver technology, Scaleway Database for Redis™ stores your data in the RAM of the underlying machine rather than on a disk (SSD/HDD). In other words, for each request to read, to insert, or to update data in a database, this can be executed using data available in the fastest and closest storage of your compute resource, the memory. Traditional databases like MySQL or PostgreSQL store data on a disk which inevitably introduces IOPs and results in latency on each operation. Redis™ is known for delivering millisecond response time and high performance for millions of requests per second to empower demanding workloads. The combination of a powerful in-memory data storage such as Redis and the management of the resources set-up, securisation, scaling and maintenance makes Scaleway Database for Redis a handy solution to improve the usability of your application 
+One of the most common ways to implement cache is storing frequently accessed data in Redis™ (therefore in memory), and serving your application's request from there, should data be available, instead of from your traditional database. If data is not available in memory, it can always be retrieved from the primary database. Below is a simplified schema explaining this process . 
+
+## How can I access my Redis once it is provisioned?
+
+You can connect to your Redis™  using the IP address.es and port.s available in the details page of your Database for Redis™ Instance next to the title **Endpoints**.
+- There are as many `IP:port` as nodes in your cluster (one if standalone).
+- If you are in cluster mode (3 nodes or more), you have to use a cluster specific connector able to connect to multiple endpoints.
+
+ADD SCREENSHOT.
+
+## What is the Private Networks feature for Database Instances?
+
+[Private Networks](https://www.scaleway.com/en/docs/network/vpc/quickstart/) allows you to configure your own Layer-2 network with IPv4 addresses on Database for Redis™ Instances. 
+You can therefore enhance the security of your system architecture by isolating it from the Internet. It also increases the security of your databases, since Instances in your Private Network can directly be connected to your Redis™ Instance, without passing through the public Internet.
+- In standalone mode, you can create new Redis™ Instances to attach to your Private Network, or attach existing Database Instances by adding Private Networks endpoints to them. 
+- In standalone mode, you can attach Private Networks to your Redis™ Instances at the creation level or add a Private Networks endpoint to your existing Redis™ Instance from the console.
+
+<Message type="note">
+
+Bear in mind that in cluster mode, Private Network is only available at the creation of the cluster Instance and cannot be modified afterwards. If you want to switch back to a public network or update your IP adresses, you have to create a new Redis Instance. 
```

Comment:
Sometime Private Network is singular, others plural
I'd say `you will have to create a new RedisTM Instance`